pkg/interop/runtime/runtime.go

```diff
@@ -59,3 +59,11 @@ func Verification() byte {
 func GasLeft() int64 {
 	return 0
 }
+
+// GetNotifications returns notifications emitted by contract h.
+// 'nil' literal means no filtering. It returns slice consisting of following elements:
+// [  scripthash of notification's contract  ,  emitted item  ].
+// TODO update to return slice of pointers to structs after #1032
```

Comment:
Not sure this TODO is useful here, better add a comment to #1032 and tell which syscall is being used here.

Reply:
Fixed.